.s2i/httpd-cfg/01-commercial.conf

```
@@ -178,6 +178,12 @@ AddType text/vtt                            vtt
    # Redirect for renamed Ingress Controller document
     RewriteRule ^container-platform/(4\.1[2-6])/networking/nw-ingress-controller-endpoint-publishing-strategies\.html$ /container-platform/$1/networking/nw-configuring-ingress-controller-endpoint-publishing-strategy.html [NE,R=302,L]
 
+    # Redirect for Networking operators move per https://github.com/openshift/openshift-docs/pull/85085
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/aws_load_balancer_operator/(.*)? /container-platform/$1/networking/networking_operators/aws_load_balancer_operator/$2 [NE,R=302]
+    RewriteRule ^container-platform/(4\.1[6-8])/networking/ingress-operator.html /container-platform/$1/networking/networking_operators/ingress-operator.html [NE,R=302]
+    RewriteRule ^container-platform/(4\.17|4\.18])/networking/network_security/ebpf_manager/(.*)? /container-platform/$1/networking/networking_operators/ebpf_manager/$2 [NE,R=302]
+    RewriteRule ^container-platform/(4\.17|4\.18])/networking/external_dns_operator/(.*)? /container-platform/$1/networking/networking_operators/external_dns_operator/$2 [NE,R=302]
```

```suggestion
    RewriteRule ^container-platform/(4\.17|4\.18)/networking/network_security/ebpf_manager/(.*)? /container-platform/$1/networking/networking_operators/ebpf_manager/$2 [NE,R=302]
    RewriteRule ^container-platform/(4\.17|4\.18)/networking/external_dns_operator/(.*)? /container-platform/$1/networking/networking_operators/external_dns_operator/$2 [NE,R=302]
```
Just peeping due to the Slack thread. If I'm not in too much of an afternoon lull, these brackets can go.

---

You're quite right, good looking out!